overlord/devicestate/firstboot20_test.go

```diff
@@ -512,6 +514,14 @@ func (s *firstBoot20Suite) TestPopulateFromSeedCore20RecoverMode(c *C) {
 }
 
 func (s *firstBoot20Suite) TestLoadDeviceSeedCore20(c *C) {
+	var calledcreateAllUsers = false
+	r := devicestate.MockCreateAllSystemUsers(func(state *state.State, assertDb asserts.RODatabase, model *asserts.Model, serial *asserts.Serial, sudoer bool) ([]*devicestate.CreatedUser, error) {
+		calledcreateAllUsers = true
+		var createdUsers []*devicestate.CreatedUser
+		return createdUsers, nil
+	})
```

Comment:
The `createdUsers` variable is nil anyway so we can just return nil. Also, it's a bit of a nitpick but we can return an error here to ensure that it's not called. Alternatively/additionally, I usually like to replace the `called*` variable and subsequent checks by a call failing the test (just a suggestion):
```suggestion
	r := devicestate.MockCreateAllSystemUsers(func(state *state.State, assertDb asserts.RODatabase, model *asserts.Model, serial *asserts.Serial, sudoer bool) ([]*devicestate.CreatedUser, error) {
	    err := errors.New("unexpected call to CreateAllSystemUsers")
        c.Error(err)
		return nil, err
	})
```

Reply:
That's a good suggestion. This is now fixed in the tests.